starter_templates/go/cmd/mygit/main.go

```diff
@@ -0,0 +1,35 @@
+package main
+
+import (
+	"fmt"
+	// Uncomment this block to pass the first stage!
+	//	"os"
+)
+
+// Usage: your_git.sh <command> <arg1> <arg2> ...
+func main() {
+	// You can use print statements as follows for debugging, they'll be visible when running tests.
+	fmt.Println("Logs from your program will appear here!")
+
+	// Uncomment this block to pass the first stage!
+	//
+	//	switch command := os.Args[1]; command {
```

Comment:
```suggestion
	// switch command := os.Args[1]; command {
```

(similar changes for other commented lines in this file. The whole "block" should be indented by `//` followed by a single space).